### DISCUSSION TITLE
两只鲨鱼

### DISCUSSION THREAD

**Discussion by Crafty-rabbit** [share121/inter-knot#1332] [其他]
这两个真的好可爱，真想都抱回家狠狠的惩罚一下口牙
![1722322134095](https://github.com/user-attachments/assets/1db01511-8e35-420f-8d21-31b588229caf)
![1723077602576](https://github.com/user-attachments/assets/9cb63252-b5ac-4d71-b128-b661ddc651a9)

**Comment by Jiucheng258** [comment]
谁家小孩

**Comment by bin-lin2** [comment]
艾莲古拉云璃三厨狂喜
